### PR TITLE
do not consider histogram cells full if the contain very little points

### PR DESCRIPTION
**Pull request by baumanta**
This got dropped somewhere in all the changes. Originally the was a minimum of points which make up a blocked histogram cell. As it was now, the cell would be blocked even if it contains just one point, which makes the algorithm very sensitive to noise.

Should be flight tested, as I'm not sure, when we lost this and if the chosen default value of 10 points is compatible with the Realsense.

### CODE REVIEW

**Comment by nicovanduijn** [comment]
Is this still relevant after the move to non-binary histogram?

**Comment by LorenzMeier** [comment]
I'm pretty sure it is, since you don't want to respond to basic noise.

File: local_planner/src/nodes/planner_functions.cpp [hunk 1]
```diff
@@ -172,7 +173,7 @@ void generateNewHistogram(Histogram& polar_histogram,
   // Normalize and get mean in distance bins
   for (int e = 0; e < GRID_LENGTH_E; e++) {
     for (int z = 0; z < GRID_LENGTH_Z; z++) {
-      if (counter(e, z) > 0) {
+      if (counter(e, z) > n_points_occupied) {
```
Comment (jkflying) [inline local_planner/src/nodes/planner_functions.cpp]: This should be `>=`, so that n_points_occupied = 1 works with a single point in the cell.

**Comment by mrivi** [comment]
Superseed by #286

**Pull request closed without merging**